tests/test_getting_started.py

```diff
@@ -258,7 +258,7 @@ def test_phase_transitions():
     # Check objective function value
     f = np.array(sol["f"])
     np.testing.assert_equal(f.shape, (1, 1))
-    # np.testing.assert_almost_equal(f[0, 0], 110875.0772043361)
+    np.testing.assert_almost_equal(f[0, 0], 110875.0772043361)
```

Comment:
Can you push (or PR) this into the #136 
Since everything else is already done

Reply:
Ok :)